compiler/testData/diagnostics/tests/thisAndSuper/unqualifiedSuper/unqualifiedSuperWithInnerClass.fir.kt

```diff
@@ -0,0 +1,33 @@
+open class A {
+    open fun foo() {}
+}
+
+interface B {
+    fun bar() {}
+}
+
+interface Q {
+    fun qux() {}
+}
+
+class C : A(), B {
+    override fun foo() {
+        super@C.foo()
+    }
+
+    override fun bar() {
+        super@C.bar()
+    }
+
+    inner class D : A(), Q {
+        override fun foo() {
+            super@C.foo()
+            super@D.foo()
+        }
+
+        override fun qux() {
+            super@C.qux()
```

Comment:
This `super@C` in FIR has an ambiguity resolve error though it's clearly not ambiguous. We should fix it in FIR.

Reply:
Ah I misunderstood the code. It's ambiguous indeed since `qux` cannot be resolved. It seems we may want to give unresolved error a higher priority than others so that unresolved error cannot be suppressed by other errors.